src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,206 @@
+import { useEffect, useState } from 'react';
+
+import { useParams, useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
 
+import { Person } from '../types';
+import { getPeople } from '../api';
+
+type PageState =
+  | 'loading'
+  | 'error'
+  | 'nothingFound'
+  | 'changeFilter'
+  | 'showTable';
+
 export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>();
+  const [filteredList, setFilteredList] = useState<Person[]>();
+  const [pageState, setPageState] = useState<PageState>('loading');
+  const [urlSlug, setUrlSlug] = useState<string>();
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const { slug } = useParams();
+
+  // Here it'll take the response from API and add father and mother to
+  // each person, if they exist...
+  const addMotherAndFather = (response: Person[]): Person[] => {
+    const peopleList = response.map((person) => {
+      const findMother = person.motherName
```

Comment:
variable name makes me think it's a function - verbs like `find` are used for functions, usually.
```suggestion
      const personMother = person.motherName
```